### PR TITLE
Removed unnecessary message in ListCommand.php

### PR DESCRIPTION
**Pull request by ImagicalGamer**
### Description

Removes the empty message sent to a console user if there are no players on the server.

### Tests & Reviews

I have indeed tested this commit and it works.

If the CommandSender is a ConsoleCommandSender and the online player count is 0 it will no longer write an empty message to the console.

### CODE REVIEW

**Comment by ImagicalGamer** [comment]
I'm squashing the commits.

File: src/pocketmine/command/defaults/ListCommand.php [hunk 1]
```diff
@@ -51,10 +52,13 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 				++$onlineCount;
 			}
 		}
-
+		
 		$sender->sendMessage(new TranslationContainer("commands.players.list", [$onlineCount, $sender->getServer()->getMaxPlayers()]));
+		if(($sender instanceof ConsoleCommandSender) and ($onlineCount === 0)){
```
Comment (ishitatsuyuki) [inline src/pocketmine/command/defaults/ListCommand.php]: Redundant. If it's a player, the count will be positive.

File: src/pocketmine/command/defaults/ListCommand.php [hunk 2]
```diff
@@ -51,10 +52,13 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 				++$onlineCount;
 			}
 		}
-
+
```
Comment (ishitatsuyuki) [inline src/pocketmine/command/defaults/ListCommand.php]: Formatting.

**Comment by dktapps** [comment]
Unnecessary.

**Pull request closed without merging**